frontend/src/metabase/admin/people/components/MembershipSelect/MembershipSelect.styled.tsx

```diff
@@ -0,0 +1,51 @@
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+
+export const MembershipSelectContainer = styled.ul`
+  padding: 0.5rem 0;
+  width: 300px;
+  max-height: 600px;
+`;
+
+interface MembershipSelectItemProps {
+  isDisabled?: boolean;
+}
+
+export const MembershipSelectItem = styled.li<MembershipSelectItemProps>`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: ${props => (props.isDisabled ? "unset" : "pointer")};
+  padding: 0.5rem 1.5rem;
+  background-color: ${color("white")};
+  color: ${color("text-medium")};
+  font-weight: 700;
+
+  ${props =>
+    !props.isDisabled &&
+    css`
+      &:hover {
+        color: ${color("white")};
+        background-color: ${color("accent7")};
+
+        .Icon {
```

Comment:
You can do something like this:

instead of

```
export default Icon;
```

use

```
export default Object.assign(Icon, {
   Root: StyledIcon
});
```

and then use `${Icon.Root}` in styles. At least this is the approach we use for other similar cases. WDYT?

Reply:
@ranquild thanks for the clarification, I misunderstood it at first. Will update the branch!